### ISSUE TITLE
Actions icon looks like Sun icon

### ISSUE THREAD

**Issue by jensy** [grommet/grommet-icons#190]
We got feedback that the Actions gear icon looks too much like a sun, I tend to agree.

<img width="1188" alt="image" src="https://user-images.githubusercontent.com/105723/87589419-70576a00-c6a2-11ea-97a4-41ae2c783e84.png">

We to update it to make it look more like a gear or an alternative idea.

**Comment by L0ZZI** [comment]
yeah, I had a rev of the settings that separates the two more. The `solaris` icon is a brand and I wonder if we deprecate that in the next release.
